hoptimator-kafka-adapter/src/main/java/com/linkedin/hoptimator/catalog/kafka/KafkaTopicAcl.java

```diff
@@ -0,0 +1,13 @@
+package com.linkedin.hoptimator.catalog.kafka;
+
+import com.linkedin.hoptimator.catalog.Resource;
+
+public class KafkaTopicAcl extends Resource {
```

Comment:
Can we add some javadoc on what this class is for? By reading just the constructor I am not able to understand what it actually is for.

Reply:
I'll mark these private. They aren't really part of the API.